### PR TITLE
feat(dedicated): integrate Support module

### PR DESCRIPTION
**Pull request by FredericEspiau**
Now, the dashboard side menu is handled by the main dashboard and not by the `userAccount` part
Signed-off-by: Frédéric Espiau <frederic.espiau@corp.ovh.com>

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/app.js [hunk 1]
```diff
@@ -30,10 +29,12 @@ import ovhManagerBanner from '@ovh-ux/manager-banner';
 import ovhManagerEnterpriseCloudDatabase from '@ovh-ux/manager-enterprise-cloud-database';
 import ovhManagerNavbar from '@ovh-ux/manager-navbar';
 import ovhManagerServerSidebar from '@ovh-ux/manager-server-sidebar';
+import ovhManagerSupport from '@ovh-ux/manager-support';
```
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/app.js]: Global app

File: packages/manager/apps/dedicated/client/app/account/account.routing.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    {
+      name: 'app.account.otrs-ticket',
+      redirectTo: 'support',
+      url: '/ticket',
+    },
+    {
+      name: 'app.account.otrs-ticket-details',
+      redirectTo: 'support.tickets.ticket',
+      url: '/ticket/:ticketId',
```
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/account/account.routing.js]: You want me to create a folder for two redirections ? Isn't it a tad bit exaggerated ? 🤔

File: yarn.lock [hunk 3]
```diff
@@ -11859,7 +11859,7 @@ ovh-ngstrap@^4.0.2:
   resolved "https://registry.yarnpkg.com/ovh-ngstrap/-/ovh-ngstrap-4.0.2.tgz#059291bfb9f59b78df991e476dd2ede0507cb3c9"
   integrity sha1-BZKRv7n1m3jfmR5HbdLt4FB8s8k=
 
-ovh-ui-angular@^3.9.6:
+ovh-ui-angular@^3.9.5, ovh-ui-angular@^3.9.6:
```
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply yarn.lock]: I won't do this in this PR, it's not its purpose :)

File: packages/manager/apps/dedicated/client/app/account/user/emails/user-emails.html [hunk 4]
```diff
@@ -5,7 +5,7 @@
         <p class="my-5">
             <span data-translate="user_account_email_info"></span><br />
             <span data-translate="user_account_email_assitance_info"></span>
-            <a href="#/ticket"
+            <a href="#/support"
```
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/account/user/emails/user-emails.html]: The state is held by a module :(

File: yarn.lock (hunk 3 above)
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply yarn.lock]: My module and app are kept up to date :)

File: packages/manager/apps/dedicated/package.json [hunk 5]
```diff
@@ -136,7 +136,7 @@
     "validator": "^3.39.0"
   },
   "devDependencies": {
-    "@ovh-ux/manager-webpack-config": "^3.1.1",
+    "@ovh-ux/manager-webpack-config": "^3.2.2",
```
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/package.json]: This was noise, I've reverted the change :)

**Comment by ovh-cds** [comment]
CDS Report build-monorepository#6387.0 ✘  
* 
  * Build ✘

File: packages/manager/apps/dedicated/client/app/account/account.routing.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/account/account.routing.js]: As seen IRL, this is not a viable long term strategy :wink:

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_us#6401.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_us#6401.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_ca#6401.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_us#6521.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_eu#6521.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_ca#6521.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_eu#6521.1 ■   
* deploy_uxci
  * deploy_uxci ■

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_us#6521.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_ca#6521.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Pull request merged**